### PR TITLE
[SignalR] Client feature matrix update

### PR DESCRIPTION
**Pull request by anurse**
Trying another format that includes version numbers.

[Internal Review Link](https://review.docs.microsoft.com/en-us/aspnet/core/signalr/client-features?view=aspnetcore-3.0&branch=anurse%2Fmore-signalr-matrix)

### CODE REVIEW

File: aspnetcore/signalr/client-features.md [hunk 1]
```diff
@@ -12,19 +12,19 @@ uid: signalr/client-features
...
-| Automatic Reconnection ([.NET](/aspnet/core/signalr/dotnet-client?view=aspnetcore-3.0&tabs=visual-studio#handle-lost-connection), [JavaScript](/aspnet/core/signalr/javascript-client?view=aspnetcore-3.0#reconnect-clients))          |✔|✔| |
-| WebSockets Transport |✔|✔|✔|
-| Server-Sent Events Transport |✔|✔| |
-| Long Polling Transport |✔|✔|✔|
-| JSON Hub Protocol |✔|✔|✔|
-| MessagePack Hub Protocol |✔|✔| |
+| Azure SignalR Service Support |1.0.0|1.0.0|1.0.0|
+| [Server-to-client Streaming](xref:signalr/streaming)          |1.0.0|1.0.0|1.0.0|
+| [Client-to-server Streaming](xref:signalr/streaming)          |3.0.0|3.0.0|3.0.0|
+| Automatic Reconnection ([.NET](/aspnet/core/signalr/dotnet-client?view=aspnetcore-3.0&tabs=visual-studio#handle-lost-connection), [JavaScript](/aspnet/core/signalr/javascript-client?view=aspnetcore-3.0#reconnect-clients))          |3.0.0|3.0.0|❌|
```
Comment (anurse) [inline aspnetcore/signalr/client-features.md]: Not sure about the ❌. It doesn't line up great with the version numbers in GitHub. We'll see how the final review version comes out. I can continue to leave it blank if we think that would be clearer.

Comment (anurse) [reply aspnetcore/signalr/client-features.md]: Looks like the alignment is OK, so I think we can keep it:

image

Comment (halter73) [reply aspnetcore/signalr/client-features.md]: We Should probably change the `monikerRange: '>= aspnetcore-3.0'` up top.

Comment (bradygaster) [reply aspnetcore/signalr/client-features.md]: I **really** like it. thanks!

**Comment by scottaddie** [comment]
Thanks for the PR, @anurse!

**Pull request merged**